docker/Dockerfile.auto

```diff
@@ -3,14 +3,26 @@ FROM pytorch/pytorch:2.1.0-cuda11.8-cudnn8-runtime
 # Set the working directory in the container
 WORKDIR /usr/src/app
 
+# Install build tools and compilers
+# Avoid user interaction with tzdata when installing
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Install build tools and compilers
+RUN apt-get update && apt-get install -y \
+  build-essential \
+  cmake \
```

Comment:
cmake? Ok you made the llama-cpp as optional so you dont have to build anything when running 'auto' mode

Reply:
fxied; removed